pyorbital/orbital.py

```diff
@@ -917,7 +921,25 @@ def propagate(self, utc_time):
         return kep
 
 
+def _get_tz_unaware_utctime(utc_time):
+    """Return timzone unaware datetime object.
+
+    The input *utc_time* is either a timezone unaware object assumed to be in
+    UTC, or a timezone aware datetime object in UTC.
+    """
+    if isinstance(utc_time, datetime):
+        if utc_time.tzinfo and utc_time.tzinfo != pytz.utc:
+            raise AttributeError("UTC time expected! Parsing a timezone aware datetime object requires it to be UTC!")
```

Comment:
I think this should be a `ValueError`. The `AttributeError` I think is more to say you tried to access an attribute that didn't exist. `ValueError` (in my opinion) is more like "you gave me a value that doesn't make sense".